src/main/java/com/fabriccommunity/spookytime/block/CraftingFluidBlock.java

```diff
@@ -0,0 +1,69 @@
+package com.fabriccommunity.spookytime.block;
+
+import net.minecraft.block.BlockState;
+import net.minecraft.block.FluidBlock;
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.ItemEntity;
+import net.minecraft.fluid.BaseFluid;
+import net.minecraft.item.ItemConvertible;
+import net.minecraft.item.ItemStack;
+import net.minecraft.recipe.Ingredient;
+import net.minecraft.sound.SoundCategory;
+import net.minecraft.sound.SoundEvent;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public abstract class CraftingFluidBlock extends FluidBlock {
+	public final SoundEvent craftingSound;
```

Comment:
`public static final SoundEvent CRAFTING_SOUND;` or death!

Reply:
i can't make it static, it's defined in an initializer

Reply:
so death i guess